### PR TITLE
Owners and Governance docs updates

### PR DESCRIPTION
**Pull request by tsurdilo**
* Adds Mauricio
* Updates Governance

### CODE REVIEW

**Comment by ruromero** [comment]
LGTM

**Comment by tsurdilo** [comment]
@cathyhongzhang can you look at the governance doc updates here to make sure its ok with you ?

File: workflow/spec/governance/readme.md [hunk 1]
```diff
@@ -7,10 +7,9 @@
 * After a review, owners can approve the changes by commenting on the PR with "LGTM" ("Looks good to me"), 
 request changes to the PR with "CR" ("Changes requested"), or reject the PR with "NA" ("Not acceptable").
 * Owners must provide an explanation for both "CR" and "NA" comments to the contributor.
-* 2 "LGTM" comments by different owners are needed in order for a change to be accepted.
-* If the contributor is an owner, then only 1 "LGTM" vote is needed.
+* 2 "LGTM" and no "NA" comments are needed in order for a change to be accepted.
```
Comment (cathyhongzhang) [inline workflow/spec/governance/readme.md]: Thanks for checking with me!  To make it clearer, I would like to suggest a change to 2 "LGTM" from owners other than the contributor and no "NA" comments are needed in order for a change to be accepted. Does this make sense to you?

Comment (tsurdilo) [reply workflow/spec/governance/readme.md]: updated. let me know if this is ok with you now

File: workflow/spec/governance/readme.md [hunk 2]
```diff
@@ -7,10 +7,9 @@
 * After a review, owners can approve the changes by commenting on the PR with "LGTM" ("Looks good to me"), 
 request changes to the PR with "CR" ("Changes requested"), or reject the PR with "NA" ("Not acceptable").
 * Owners must provide an explanation for both "CR" and "NA" comments to the contributor.
-* 2 "LGTM" comments by different owners are needed in order for a change to be accepted.
-* If the contributor is an owner, then only 1 "LGTM" vote is needed.
+* 2 "LGTM" votes (from owners other than the contributor) and no "NA" votes are needed in order for a change to be accepted.
 * If there is an existing "LGTM" vote by a owner and another owner comments with "NA", then
-the owners must work together to reach a concensus on the requested changes (preferably publicly including the contributor).
+the owners must work together to reach a consensus on the requested changes (preferably publicly including the contributor).
```
Comment (cathyhongzhang) [inline workflow/spec/governance/readme.md]: LGTM. Thanks!

**Pull request merged**